### PR TITLE
Music Releases 🎶 - Sofia & Ingela

### PR DESCRIPTION
**Pull request by sofiavazs**
**Link to my deployed project:**
https://musicreleases-ingela-sofia.netlify.app/

**My reflections on how this week's project turned out:**
_Ingela_
It was fun to start working in React. It seem like a smart and structured way of working. I think our music releases site turned out really good. It looks like the provided the design and have all the other requirements of the blue level.

_Sofia_
Really enjoyed making this project, specially in pair-programming! It was super fun to start with React and as Ingela mentioned it does seem a really nice and structured way of working, the components concept really makes it easier to understand  because it's smaller blocks of code. I'm really happy how the project turned out, we managed to comply with the required design and the variation of hover effects. 

**Things I'd like to have clarified or explained in more detail:**
_Ingela_ & _Sofia_
Not really! The material and lectures have been great and relevant for the project and the size of the project was perfect.

### CODE REVIEW

File: code/src/components/Artist.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+export const Artist = (props) => {
+    return (
+        <div className='artist-name'>
+            {
+                props.item.artists.map((artist) => {
+                    return <a 
+                            key={artist.id}
+                            href={artist.external_urls.spotify}>
```
Comment (EmOlsz) [inline code/src/components/Artist.js:11]: Wrong indents, you seem to forgot about parentheses which can wrap returned component

File: code/src/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+export const Header = () => { 
+    return <header>
+            <h1>New albums & singles</h1>
+            <hr></hr>
```
Comment (EmOlsz) [inline code/src/components/Header.js:6]: Same here!

File: code/src/index.css [hunk 3]
```diff
@@ -11,3 +12,152 @@ code {
...
+.artist-name {
+  color: #a0a0a0;
+  font-size: 14px;
+  font-family: Helvetica, sans-serif;
+  margin-left: 15px;
+  padding-bottom: 5px;
+}
+
+.album-title {
+  color: #ffffff;
```
Comment (EmOlsz) [inline code/src/index.css:51]: `color: #fff`

File: code/src/index.css [hunk 4]
```diff
@@ -11,3 +12,152 @@ code {
...
+}
+
+.album-title:hover {
+  color: #ffffff;
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+.artist-name a:last-child span { /* to prevent commas being added even if it's only one artist*/
+  display: none
```
Comment (EmOlsz) [inline code/src/index.css:139]: Clever :)

**Review by EmOlsz — APPROVED** [review]
Good work Sofia & Ingela! You've divided your components in a descriptive, clean way :)
Take a look on some of my comments, there are things that you can improve, nonetheless they are not blockers. Keep it this way :)

File: code/src/App.js [hunk 5]
```diff
@@ -1,12 +1,12 @@
 import React from 'react'
-import data from './data.json'
+import { Header } from './components/Header'
```
[the remark this answers is not in the document]
Comment (sofiavazs) [reply code/src/App.js:2]: Thank you! 🙌 this is really helpful to know so I can change it in my current project!

**Pull request closed without merging**